updating/update-using-custom-machine-config-pools.adoc

```diff
@@ -0,0 +1,90 @@
+[id="update-using-custom-machine-config-pools"]
+= Performing a rolling update 
+include::modules/common-attributes.adoc[]
+:context: update-using-custom-machine-config-pools
+
+toc::[]
+
+
+Using a rolling update strategy allows you to update groups of nodes in a cluster one-by-one, test the nodes and the applications on those nodes, and if everything works as you expect, you can proceed to update another group of nodes. 
+
+The {product-title} update process is highly-available by design and there are many features, such as pod disruption budgets, pod affinity, health checks, and replicas, which make the applications highly-available in case of node failures.
+
+However there might be some secenarios where you want a more controlled rollout of an upgrade to the worker nodes in order to ensure that mission-critical applications stay available during the whole update, even if the upgrade process causes a failure. After the cluster is updated, you can update those nodes at more appropriate time.
```

Comment:
This is confusing as we are talking about `after the cluster is updated` (which includes updating all the nodes) and then we are asking users to update the nodes later.